### PR TITLE
#167313399 User should be able to sign out

### PR DESCRIPTION
**Pull request by minega25**
#### What does this PR do?
- User should be able to sign out
#### Description of Task to be completed?
- 
#### How should this be manually tested?
- `git checkout -b ft-user-sign-out-167313399`
- `git pull origin ft-user-sign-out-167313399`
- `npm install`
- create a `.env` file or export the following  environment variables 

[ API_VERSION=v1, BASE_URL=localhost, PORT=5000,DB_PORT=5432]  
- `npm run dev`
- Using postman test using the following endpoints

1. /api/v1/users/signout -- include a logged in user token to be signed out
2. Test any other protected endpoint to check if user is signed out

#### What are the relevant pivotal tracker stories?
[#167313399](https://www.pivotaltracker.com/story/show/167313399)
#### Screenshots (if appropriate)
Screen Shot 2019-08-07 at 23 06 08
Screen Shot 2019-08-08 at 00 14 48
Screen Shot 2019-08-08 at 00 33 40

#### Questions:
- What is the best practice for signing out users as the industry prefers doing it from the client side.

### CODE REVIEW

File: test/users.test.js [hunk 1]
```diff
@@ -74,7 +74,7 @@ describe('Users', () => {
       })
       .set('Accept', 'Application/JSON')
       .end((error, res) => {
-        adminToken = `Bearer ${res.body.token}`;
+        adminToken = `${res.body.token}`;
```
Comment (Mnickii) [inline test/users.test.js]: why did we remove Bearer?

File: .eslintrc.js [hunk 2]
```diff
@@ -6,6 +6,9 @@ module.exports = {
     es6: true,
     mocha: true
   },
+  parserOptions: {
```
Comment (Mnickii) [inline .eslintrc.js]: should be ``` "parserOptions": {
        "sourceType": "module",
    }```

**Review by Mnickii — CHANGES_REQUESTED** [review]

File: .eslintrc.js (hunk 2 above)
Comment (minega25) [reply .eslintrc.js]: I think my auto formatter in vscode changed this. Let me work on it

File: test/users.test.js (hunk 1 above)
Comment (minega25) [reply test/users.test.js]: Let me work on this as well

Comment (minega25) [reply test/users.test.js]: @Mnickii I am done working on this

File: .eslintrc.js (hunk 2 above)
Comment (minega25) [reply .eslintrc.js]: @Mnickii I have worked on this.

**Review by nshutijonathan — APPROVED** [review]

**Review by Mnickii — APPROVED** [review]
LGTM

**Review by salviosage — APPROVED** [review]
LGTM

**Review by habinezadalvan — APPROVED** [review]

**Pull request merged**